spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -981,6 +989,22 @@ def notify_close(self):
             return params
 
     # -------------------------------------------------------------------------
+    def set_debug_panel(self, debug_panel, language):
+        """Enable/disable debug panel"""
+        debugger_panel = self.panels.get(DebuggerPanel)
+        if language == 'py':
+            debugger_panel.setVisible(True)
+        else:
+            debugger_panel.setVisible(False)
+
+    def set_folding_panel(self, folding, language):
+        """Enable/disable debug panel"""

```

Comment:
The comment should be something like 
```python
"""Enable/disable folding panel"""
```

Reply:
Per PEP 257, docstrings must be sentences ending in a period.